### ISSUE TITLE
Correct child benefit for single parents

### ISSUE THREAD

**Issue by Eric-Sommer** [iza-institute-of-labor-economics/gettsim#158]
### Current and desired situation

Currently, single parents receive the full child benefit (Kindergeld) but only half the child tax allowance (Kinderfreibetrag). This is wrong at least for those receiving alimony payments for their child(ren).

### Proposed implementation

Those single parents should instead receive only half the child benefit amount. If they receive child alimony *advance* (Unterhaltsvorschuss), however, they are paid the full amount.

**EDIT:** We probably have to fix two things:

- [ ] For the *Günstigerprüfung* (Kindergeld versus Kinderfreibetrag), only half the child benefit is considered, because the other half is (virtually) paid to the other parent.
- [ ] Alimony payments would increase if child allowance hits. We therefore might need to increase alimonies to avoid jumps in disposable income.

**Comment by hmgaudecker** [comment]
Do you have a reference for this? Should be quick to implement then.

**Comment by Eric-Sommer** [comment]
After a lengthy discussion with @mjbloemer , the follwing seems to emerge:

1. Single parents (SP) get the full child allowance *if* the [parent does not or does not need to pay alimonies](https://www.familienratgeber.de/rechte-leistungen/staatliche-hilfen/kinderfreibetrag.php). This seems to be equivalent to receiving alimony advance (*Unterhaltsvorschuss*).
2. If the SP receives 'proper' child alimonies, he/she receives half the child allowance. At face value, this would mean that there is a loss in disp. income if the higher child allowance hits. Alimony plays a role here as well. From the alimony payment, half the child benefit is deducted. If this child benefit vanishes, the deduction should vanish as well, creating a smooth budget line. Whether this last bit is true we're not sure.
3. To make things even more complicated, there might be different rules concering transferability of the two components of child allowance (*sächliches Existenzminimum* and *Betrag für Erziehung und Ausbildung*)
4. Sidenote: We should in principle not mistake [mother + child + father somewhere else] with [mother + child + father dead]. But this seems to be OK, since if the child does not receive enough *Halbwaisenrente*, the household may get *Unterhaltsvorschuss*

**Comment by Eric-Sommer** [comment]
Just citing from the income tax form:

> **Übertragung des Kinderfreibetrags / des Freibetrags für den Betreuungs- und Erziehungs- oder Ausbildungsbedarf
Zeile 43 bis 48**
> Bei geschiedenen oder dauernd getrennt lebenden Eltern sowie bei Eltern nichtehelicher Kinder kann ein Elternteil in der Zeile 43 beantragen, dass der Kinderfreibetrag des anderen Elternteils auf ihn übertragen wird, wenn er, nicht aber der andere Elternteil, seine Unterhaltsverpflichtung gegenüber dem Kind für 2020 zu mindestens 75 Prozent erfüllt hat oder der andere Elternteil mangels Leistungsfähigkeit nicht unterhaltspflichtig ist und keine Unterhaltsleistungen nach dem Unterhaltsvorschussgesetz gezahlt worden sind. Dies führt auch zur Übertragung des Freibetrags für den Betreuungs- und Erziehungs- oder Ausbildungsbedarf. Nachweise über die konkrete Höhe der Unterhaltsverpflichtung des anderen Elternteils sowie seine tatsächlichen Unterhaltsleistungen sind nur auf Anforderung des Finanzamts einzureichen (zum Beispiel Scheidungsurteil, Zahlungsbelege). Eine einvernehmliche Übertragung des Kinderfreibetrags ist nicht möglich.
Bei geschiedenen oder dauernd getrennt lebenden Eltern sowie bei Eltern nichtehelicher Kinder kann ein Elternteil abweichend vom Kinderfreibetrag die Übertragung des halben Freibetrags für den Betreuungs- und Erziehungs- oder Ausbildungsbedarf des anderen Elternteils in der Zeile 45 beantragen, sofern das minderjährige Kind bei dem anderen Elternteil nicht gemeldet war.
Dem Antrag auf Übertragung kann nicht entsprochen werden, wenn der Elternteil, bei dem das Kind nicht gemeldet war, Kinderbetreuungskosten trägt oder das Kind regelmäßig in einem nicht unwesentlichen Umfang betreut (zum Beispiel regelmäßig an Wochenenden und in den Ferien) und dieser Elternteil der Übertragung des Freibetrags widerspricht.
Der Kinderfreibetrag und der Freibetrag für den Betreuungs- und Erziehungs- oder Ausbildungsbedarf kann auch auf einen Stief- oder Großelternteil übertragen werden, wenn sie das Kind in ihren Haushalt aufgenommen haben. Eine Übertragung auf einen Großelternteil ist außerdem möglich, wenn dieser gegenüber dem Kind unterhaltspflichtig ist. Die Übertragung kann auch mit Zustimmung des / der berechtigten Elternteils / Elternteile erfolgen. Verwenden Sie in diesem Fall bitte die Anlage K(Papierformular mit eigenhändiger Unterschrift ist erforderlich); eine monatsweise Übertragung der Freibeträge ist insoweit nicht möglich.
Steht dem übertragenden Elternteil der Kinderfreibetrag in voller Höhe zu (zum Beispiel weil der andere Elternteil verstorben ist), wählen Sie in Zeile 47 den Wert „Zustimmung beider Elternteilen liegen vor" aus.

**Comment by mjbloemer** [comment]
A few notes and resources as discussed:

We should calculate ESt I and ESt II and not set Kindergeld to 0. Follow:

![d0630328-001](https://user-images.githubusercontent.com/31305679/143849719-2efb0665-c3ee-4718-9a31-2907b3987be2.gif)

https://www.bundesfinanzministerium.de/Content/DE/Downloads/Broschueren_Bestellservice/2020-01-30-datensammlung-zur-steuerpolitik-2019.pdf?__blob=publicationFile&v=9

- page 36: Besteuerungsschema. Kindergeld wird am Ende für die festzusetzende ESt hinzugerechnet.
- page 53ff has data on Kindergeld and Kinderfreibetrag. Kinderfreibetrag is shown as "Zusatzentlastung"

**Comment by hmgaudecker** [comment]
@MImmesberger , I'd think we fixed that at some point?

**Comment by MImmesberger** [comment]
Yes, single parents now receive only one Kinderfreibetrag instead of two. In the case of other parent not paying alimony/dead we opted for letting users overwrite `anzahl_kinderfreibeträge` instead of automatically giving the other parent 2 Freibeträge if the ID of the other parent is `-1`. I think we haven't documented this in the docstring, will do this in #805 and close this one here.
